### PR TITLE
feat:(task_repository): create tasks list data/wrong

### PR DESCRIPTION
**Pull request by kamilgadek**

### CODE REVIEW

File: packages/tasks_repository/tasks_data.dart [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class Task {
```
Comment (mszakacz) [inline packages/tasks_repository/tasks_data.dart:1]: @kamilgadek You already have a `Task` model [here](https://github.com/kamilgadek/to_do_list_kamilg/blob/main/packages/tasks_repository/lib/src/models/task.dart)
So there is no need to create it again. Please remember that the code should be **DRY** - Do Not Repeat Yourself.

What to do:
- remove lines 1 - 12 (we already have a Task model)
- adjust the tasks to a model you already have 
- create a new folder inside `tasks_repository/lib/src` named `data` and store this file there.

Let me know if any difficulties.

**Review by mszakacz — CHANGES_REQUESTED** [review]

File: packages/tasks_repository/tasks_data.dart [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+final bool isDone;
+
+Task({
+required this.title,
+required this.description,
+required this.isDone,
+});
+}
+
+List tasks = [
```
Comment (mszakacz) [inline packages/tasks_repository/tasks_data.dart:13]: One more thing, Avoid creating a public variables. Let's try to store all the variables inside a class. In this case you could create a class called TasksDevData and store the list of tasks there as a static const.

Comment (mszakacz) [inline packages/tasks_repository/tasks_data.dart:13]: Please remember to document your code. As you remember, in the real app we would take the tasks data from the server, and this list of data is for development purposes only, so let's create a comment on this (tripple slash in FLutter) and describe that it's for dev purposes only. 
To make sure that you would not forget it, you could event create a to do in  the code like that:
`// TODO(kamil): remove it later as it's for dev purposes only`

File: packages/tasks_repository/tasks_data.dart (hunk 1 above)
Comment (kamilgadek) [reply packages/tasks_repository/tasks_data.dart:1]: Ok, I made the changes. Should I do a git pull in this pull request now?

**Comment by kamilgadek** [comment]
That should be close becasuse wrong setup. The correct one is https://github.com/kamilgadek/to_do_list_kamilg/pull/18

**Pull request closed without merging**